Updates group checks from old groups to new access groups

ikirker opened the pull request:
Pretty straight-forward, mostly a search-and-replace.

ikirker committed:
Updates application modules to new access groups

ikirker committed:
Updates department-specific modules to new access groups

ikirker committed:
Unprotects group name in old note

heatherkellyucl commented on applications/gaussian/g09-c01_linda/pgi-2013.9:
Should be `ag-archpc-gaussian09` - or both like [applications/gaussian/g09-d01/pgi-2015.7](https://github.com/UCL-ARC/rcps-modulefiles/pull/7/files#diff-a7bda3514eeaf90e0e1175a40c3b6fe7b6af8120ed7eb9cbd3e56b871189e3a3)

ikirker replied:
I already added all members of the new Gaussian03 group to the Gaussian09 group, so I guess I should actually remove the old double-tests and remove all uses of the Gaussian03 group (since we have no installations of Gaussian03).

heatherkellyucl replied:
Go ahead and merge when done.

heatherkellyucl commented on dept/imperial/fhi-aims/210716_3/intel-2018/ic-wg:
The group for this should be `ag-archpc-fhiaims-imperial` or `ag-archpc-fhiaims-imperial-wolpert` if we want to label the group as well.

ikirker replied:
I guess the question is whether, if a UCL group got their own license, we would have to create a new installation or just give them access to this one.

heatherkellyucl replied:
Ah right - yes, using the same one makes sense in that case.

ikirker committed:
Removes old Gaussian03 group checks on Gaussian09

We used to have an installation of Gaussian09 installed under the old
`legg03` group because of a limit on group membership and a problem
creating new groups, but both of those problems are gone now.

Note that because the `gaussian/g09-d01/pgi-2015.7` module used to
divert people to one of two installations depending on group membership,
some users will have found themselves using a different installation
than previously. This is not expected to cause any problems, as they're
the same version of Gaussian built the same way.
